mac

```diff
@@ -132,6 +132,9 @@ else
   fancy_echo "Homebrew already installed. Skipping ..."
 fi
 
+fancy_echo "Uninstalling old Homebrew-Cask ..."
+brew uninstall --force brew-cask
```

Comment:
It'd be aesthetically pleasing to only run this if `brew-cask` is actually installed. Maybe wrap this in `if brew list | grep -Fq brew-cask`?

(This exits 0 if brew-cask is not installed, so there's no harm as-is.)